src/script/lua_api/l_object.cpp

```diff
@@ -2288,10 +2277,14 @@ luaL_Reg ObjectRef::methods[] = {
 	luamethod(ObjectRef, get_properties),
 	luamethod(ObjectRef, set_nametag_attributes),
 	luamethod(ObjectRef, get_nametag_attributes),
-	// LuaEntitySAO-only
+
 	luamethod_aliased(ObjectRef, set_velocity, setvelocity),
 	luamethod(ObjectRef, add_velocity),
 	luamethod_aliased(ObjectRef, get_velocity, getvelocity),
+	{"get_player_velocity", ObjectRef::l_get_velocity},
+	{"add_player_velocity", ObjectRef::l_add_velocity},
```

Comment:
```diff
        luamethod_aliased(ObjectRef, set_velocity, setvelocity),
        luamethod(ObjectRef, add_velocity),
+       {"add_player_velocity", ObjectRef::l_add_velocity},
        luamethod_aliased(ObjectRef, get_velocity, getvelocity),
        {"get_player_velocity", ObjectRef::l_get_velocity},
-       {"add_player_velocity", ObjectRef::l_add_velocity},
```
Otherwise it's not registered as deprecated function. It's a bit hacky, I know.